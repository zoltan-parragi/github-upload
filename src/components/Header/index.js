import React from "react";

const Header = ({ children }) => 
    <header>
	    <div className="header-container">
	        <h1>Words in the <span className="orange">Bible</span></h1>
	        <p>Search the word you choose in Bible translations.</p>
        </div>
    </header>;

export default Header;