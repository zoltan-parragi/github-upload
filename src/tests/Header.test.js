import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

describe('Header', () => {
	it('renders without crashing', () => {
	const div = document.createElement('div');
	ReactDOM.render(<Header />, div);
	ReactDOM.unmountComponentAtNode(div);
	});
	test('has valid snapshot', () => {
		const component = renderer.create(<Header />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});