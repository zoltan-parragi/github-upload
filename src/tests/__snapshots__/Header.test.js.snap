// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header has valid snapshot 1`] = `
<div
  className="header"
>
  <div
    className="header-container"
  >
    <h1>
      Words in the 
      <span
        className="orange"
      >
        Bible
      </span>
    </h1>
    <p>
      Search the word you choose in Bible translations.
    </p>
  </div>
</div>
`;
